scripts/local-cluster.sh

```diff
@@ -4,6 +4,8 @@
 
 set -euo pipefail
 
+: "${CK8S_CONFIG_PATH:?Missing CK8S_CONFIG_PATH}"
+
```

Comment:
This cannot be here, as not all commands require it.

Reply:
I thought I checked that, but I see I missed some functions. Updated now